src/main/java/gomedic/commons/core/Messages.java

```diff
@@ -33,29 +35,131 @@
      */
     public static String getSuggestions(String command) {
 
-        List<String> listOfCommands = Arrays.asList("add", "edit", "delete",
-                "clear", "find", "list", "exit", "help");
-        LevenshteinDistance stringChecker = new LevenshteinDistance();
-        List<Pair<Integer, String>> closestStrings = listOfCommands.stream()
-                .map(x -> new Pair<>(stringChecker.apply(x, command), x))
-                .sorted(Comparator.comparingInt(Pair::getKey))
-                .collect(Collectors.toList());
+        // pool of command suggestions
+        List<String> listOfCommands = Arrays.asList(
+                "help",
+                "add t/patient",
+                "view t/patient",
+                "delete t/patient",
+                "edit t/patient",
+                "list t/patient",
+                "clear t/patient",
+                "add t/doctor",
+                "view t/doctor",
+                "delete t/doctor",
+                "edit t/doctor",
+                "list t/doctor",
+                "clear t/doctor",
+                "find",
+                "add t/activity",
+                "delete t/activity",
+                "list t/activity",
+                "clear t/activity",
+                "exit");
+        // list of command targets
+        List<String> listOfTargets = Arrays.asList("t/patient", "t/doctor", "t/activity");
+        // get hashset of types for checking
+        HashSet<String> listOfTypes = new HashSet<>(listOfCommands.stream()
```

Comment:
I am slightly confused why is this called `listOfTypes` even though it contains the commands, and `listOfTargets` contains list of the types

Reply:
oh i classified the commands add, delete, edit etc as command types and the kind of data they are accessing like doctors patients etc as targets